data/transit/route/bus.json

```diff
@@ -14275,6 +14275,8 @@
         "network": "Trieste Trasporti",
         "network:wikidata": "Q3539115",
         "network:wikipedia": "it:Trieste Trasporti",
+        "old_operator": "Trieste Trasporti"
```

Comment:
I would suggest removing this line.

Reply:
Sorry, ignore the matchNames comment as I see old operator is the same name as display name.